app/models/concerns/alegre_webhooks.rb

```diff
@@ -11,21 +11,26 @@ def valid_request?(request)
     end
 
     def is_from_alegre_search_result_callback(request)
-      request.params.dig('data', 'is_shortcircuited_search_result_callback') || request.params.dig('data', 'is_search_result_callback')
+      params.dig('data', 'is_shortcircuited_search_result_callback') || params.dig('data', 'is_search_result_callback')
```

Comment:
The method receives `request` as argument but in the body you check for `params`? I think the argument name should be `params`, right?